TurkishFilter/sections/specific.txt

```diff
@@ -389,6 +389,11 @@ selamtuzla.com##.aside_ad
 kibriswebhaber.com##img[width="1100"][height="140"]
 petroturk.com##.site-content > div[class^="site-"]
 petroturk.com##div[class^="ads-"]
+unyenethaber.com##.genel > div[style^="width: 980px; height: auto;"] > a
+unyenethaber.com##.haberDetayIcerik > div[style^="margin-right: 10px; float: left;"]
+unyenethaber.com###siteSol > div[style^="width: 780px; height: auto;"]
+unyenethaber.com###siteSag > div[style^="width: 370px; height: auto;"]
+unyenethaber.com###haberAlani > div[style^="width: 100%; height: auto; overflow: hidden;"]
```

Comment:
It seems that some ads on main page are not blocked.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/103274523-31a03c80-49c2-11eb-87bf-a0ab66aec1d2.png)

</details>

Could you please block them too?

Reply:
i guess they changed something, omw